### DISCUSSION TITLE
PaddleOCR中没有参数use_npu。应该如何用PaddleOCR将模型加载昇腾NPU上

### DISCUSSION THREAD

**Discussion by wenxinmomo** [PaddlePaddle/PaddleOCR#14282] [Q&A]
from paddleocr import PaddleOCR，在这个包中，没有看到use_npu。
请问如何将模型加载昇腾NPU上，使用PaddleOCR类

![image](https://github.com/user-attachments/assets/dbd46d11-2e14-41fc-bb54-853385d544af)

**Comment by wenxinmomo** [comment]
在另一个issuces中，https://github.com/PaddlePaddle/PaddleOCR/issues/13647
看到有相关参数，是因为版本不同嘛

**Comment by GreatV** [comment]
有可能是

**Reply by wenxinmomo** [reply]
有其他信息吗

**Reply by GreatV** [reply]
用2.9版本试试

**Reply by wenxinmomo** [reply]
谢谢，我看看
+

**Reply by wenxinmomo** [reply]
么有啊，老哥。

**Reply by GreatV** [reply]
这里是有的 tools/infer/utility.py

**Reply by GreatV** [reply]
试试命令行模式

**Reply by wenxinmomo** [reply]
谢谢，我也看到了这个文件，但是就是在代码中导入，并进行初始化和推理，from paddleocr import PaddleOCR

**Reply by wenxinmomo** [reply]
我知道了，原来PaddleOCR中有引入了tools/infer/utility.py，可直接在PaddleOCR类中使用use_npu。谢谢
